src/components/drawing/index.js

```diff
@@ -678,3 +678,13 @@ drawing.setPointGroupScale = function(selection, x, y) {
 
     return scale;
 };
+
+drawing.measureText = function(tester, text, font) {
+    var dummyText = tester.append('text')
+        .text(text)
+        .call(drawing.font, font);
+
+    var bbox = drawing.bBox(dummyText.node());
+    dummyText.remove();
+    return bbox;
+};
```

Comment:
That makes sense. I didn't appreciate that the tester was also cloning it. And to add to that, it looks like I only ended up using it in one place. :/